src/windows/Link.tsx

```diff
@@ -32,6 +35,12 @@ export interface LinkState {
 }
 
 export class Link extends LinkBase<LinkState> implements FocusManagerFocusableComponent {
+    
+    //Offset to show context menu using keyboard.
```

Comment:
Nit: Please add space after "//" for consistency.

Reply:
updated